crates/storage/provider/src/traits/chain.rs

```diff
@@ -32,6 +32,11 @@ pub trait CanonStateSubscriptions: Send + Sync {
             st: BroadcastStream::new(self.subscribe_to_canonical_state()),
         }
     }
+
+    /// Subscribe to get updates about the recent pending block.
+    fn pending_blocks_stream(&self) -> watch::Receiver<Option<(SealedBlock, Vec<Receipt>)>> {
+        todo!() // Placeholder for now
+    }
```

Comment:
we want a helper type for this, this way we can expose a few additional functions that we need perhaps

Reply:
Made an alias type as for the other two subscription types
yeap?